src/uproot/models/RNTuple.py

```diff
@@ -60,8 +61,21 @@ def _keys(self):
                 keys.append(fr.field_name)
         return keys
 
-    def keys(self):
-        return self._keys
+    def keys(
+        self,
+        *,
+        filter_name=None,
+        filter_typename=None,
+        filter_branch=None,
+        recursive=False,
+        full_paths=True,
+        ignore_duplicates=False,
```

Comment:
I don't think it makes sense to have these extra arguments already if they don't do anything.

`ignore_duplicates` shouldn't exist since that's not an issue with RNTuples, and `filter_branch` also shouldn't exist since there are no "branches". The rest would be good to implement at some point.

Reply:
I did a lazy copy-paste from TTree keys() in this case, to solve missing **recursive** argument in dask.
I agree that they should be removed to avoid confusion. I will edit PR and remove the ones you mentioned.

Reply:
Ah okay, that makes sense. Then let's just remove the ones that don't make sense and leave a TODO comment for the ones that haven't been implemented.